Client/Frontend/Home/JumpBackIn/Data/JumpBackInDataAdaptor.swift

```diff
@@ -258,7 +263,9 @@ class JumpBackInDataAdaptorImplementation: JumpBackInDataAdaptor, FeatureFlaggab
         var mostRecentTab: (client: RemoteClient, tab: RemoteTab)?
 
         desktopClientAndTabs.forEach { remoteClient in
-            guard let firstClient = remoteClient.tabs.first else { return }
+            guard let firstClient = remoteClient.tabs.first else {
+                return
+            }
```

Comment:
```suggestion
            guard let firstClient = remoteClient.tabs.first else {
                return
            }
```

nit: Why the change here?